ontobio/ontol.py

```diff
@@ -598,12 +598,49 @@ def logical_definitions(self, nid):
         else:
             return []
 
+    def definition(self, nid):
+        """
+        Text definition object for a node
+
+        Arguments
+        ---------
+        nid : str
+            Node identifier for entity to be queried
+
+        Return
+        ------
+        dict
+            definition object, dict(val=TEXT, xrefs=LIST)
+        """
+        return self._get_meta_prop(nid, 'definition')
+        
+    def definition_val(self, nid):
```

Comment:
Oh it's merged already? Where at? I was thinking we would release after this PR to get my logging changes in and this. 

Oh, I just saw that it's just a dictionary that's being returned and not a real Object. I guess my suggestion doesn't make as much sense then.